src/modules/flow/http-client/http-client.c

```diff
@@ -422,13 +476,29 @@ common_get_process(struct sol_flow_node *node, void *data, uint16_t port,
     }
 
     sol_http_params_init(&params);
-    if (mdata->accept && !sol_http_param_add(&params,
-        SOL_HTTP_REQUEST_PARAM_HEADER("Accept", mdata->accept))) {
-        SOL_WRN("Failed to set query params");
-        r = -ENOMEM;
-        goto err;
+
+    if (mdata->accept) {
+        struct sol_vector priorities;
+        struct sol_http_content_type_priority *pri;
+
+        if (!sol_http_param_add(&params,
+            SOL_HTTP_REQUEST_PARAM_HEADER("Accept", mdata->accept))) {
+            SOL_WRN("Failed to set the 'Accept' header with value: %s",
+                mdata->accept);
+            r = -ENOMEM;
+            goto err;
+        }
+
+        r = sol_http_parse_content_type_priorities(sol_str_slice_from_str(mdata->accept), &priorities);
```

Comment:
you're parsing your own accept. You should get the accept from the HTTP request. In this case, the `GET` port, you should do the request with `mdata->accept` and wait for the server to reply with its `Content-Type: text/stream`, then, at http-client-request callback you can decide if you'll follow with SSE or not.